lib/src/hmssdk.dart

```diff
@@ -219,22 +219,25 @@ class HMSSDK {
   /// The [hmsActionResultListener] informs about whether the message was successfully sent, or the kind of error if not.
   void sendGroupMessage(
       {required String message,
-      required String roleName,
+      required List<HMSRole> roles,
       String? type,
       HMSActionResultListener? hmsActionResultListener}) async {
-    var arugments = {"message": message, "type": type, "role_name": roleName};
+    List<String> rolesMap = [];
+    roles.forEach((role) => rolesMap.add(role.name));
+
+    var arguments = {"message": message, "type": type, "roles": roles};
     var result = await PlatformService.invokeMethod(
         PlatformMethod.sendGroupMessage,
-        arguments: arugments);
+        arguments: arguments);
```

Comment:
`rolesMap` should be sent here instead of `roles`?

Reply:
To platform method, list of role name strings will be sent.
To the action result listener back to the app, list of HMSRole objects is to be sent

Reply:
roles are list of HMSRole
rolesMap is list of String